apps/src/lib/kits/maker/boards/microBit/LightSensor.js

```diff
@@ -67,11 +82,39 @@ export default class LightSensor extends EventEmitter {
     this.board.mb.stopStreamingAnalogChannel(sensor_channels.lightSensor); // disable light sensor
   }
 
-  //TODO
-  getAveragedValue() {}
+  // Get the averaged value over the give range of ms, adjusted within specified range
```

Comment:
I see a given range (as the function parameter) but where is the specified range?